NDTensors/src/tensor/tensor.jl

```diff
@@ -107,6 +107,10 @@ function Tensor(datatype::Type{<:AbstractArray}, inds::Tuple)
   return Tensor(generic_zeros(datatype, dim(inds)), inds)
 end
 
+function Tensor()
+  return Tensor(Zeros{default_eltype(),1,default_datatype(default_eltype())}(()), ())
```

Comment:
Is the plan to use `default_storagetype` for this kind of thing? Also seems like this could just be defined as: `Tensor() = Tensor(())`.

Reply:
This type of definition might be overkill, since I am simply calling the default function, I was using this as a proof of principle to test if the `Zeros` type was working.

Reply:
Ok, no problem. Definitely understand this is a draft where you are trying different things out and it's not in a final state, I can hold off on reviewing for now and you can let me know when you think it is ready for review.